src/main/java/com/stripe/model/StripeObject.java

```diff
@@ -26,10 +28,19 @@ public String toString() {
 				PRETTY_PRINT_GSON.toJson(this));
 	}
 
+	public StripeResponse getResponse() {
+		return response;
+	}
+	public void setResponse(StripeResponse response) {
+		this.response = response; 
+	}
+
 	public String toJson() {
 		return PRETTY_PRINT_GSON.toJson(this);
 	}
 
+	private StripeResponse response;
```

Comment:
We should also ensure that the `response` attribute is not included in the JSON serialization.

Reply:
Changed to

```java
private transient StripeResponse response;
```
To prevent serialization of this field